Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -902,6 +903,32 @@ def audit_revision_and_version_scheme
       end
     end
 
+    def audit_livecheck
+      return unless formula.livecheckable?
+      return unless @online
+
+      begin
```

Comment:
I think the better way is to abstract the printing out of the `livecheck_formulae` function so you can just check if any data is returned instead of jumping through a bunch of hoops like this.

In my ideal world this would just run
```ruby
problem "" if Homebrew::Livecheck.livecheck_formulae([formula]).nil
```

---

You might also just want to skip the audit for this round

Reply:
Since those changes should probably be part of another PR, I guess I'll remove the audit for now.

Reply:
Sounds good to me, better to do small iterations.

Reply:
Agreed on pulling this into another PR 👍🏻